### ISSUE TITLE
It does not work fine on codeforces.com.

### ISSUE THREAD

**Issue by scalpelx** [philc/vimium#2035]
The icon on the toolbar become gray. And on some webpages, such as [http://codeforces.com/problemset/problem/635/A](http://codeforces.com/problemset/problem/635/A), it doesn't work.
OS version:Windows 10 x64
Vimium Version:1.54

**Comment by smblott-github** [comment]
Thanks, @scalpelx.  Vimium is working fine on that site for me.  Is Vimium working on *any* sites for you?

Might you have added an exclusion rule accidentally?  You can check (and remove) your exclusion rules on the options page.

**Comment by scalpelx** [comment]
@smblott-github I'm not sure that vimium is working fine on all sites. I have tested many sites and found that only page is not working well. It doesn't in the exclusion rule.

**Comment by scalpelx** [comment]
@smblott-github And there is another problem. Vimium's icon is gray on all sites despite It works well on that site.

**Comment by tsiemens** [comment]
I can also confirm the issue with the greyed icon on all sites, on this version of Chrome and Vimium, using the rpm packaged version of Chrome (Fedora). Vimium itself is working though.

I see this in the background task's logs:
```
extensions::lastError:133 Unchecked runtime.lastError while running browserAction.setIcon: Icon invalid.reportIfUnchecked @ extensions::lastError:133
main.js:769 checking keyQueue: [ g ]
main.js:771 new KeyQueue: 1g
main.js:769 checking keyQueue: [ 1gt ]
main.js:771 new KeyQueue: 
```

**Comment by smblott-github** [comment]
OK.  Thanks for the reports, folks.  #2040 fixes this.

**Comment by lonelam** [comment]
It seems that vimium still doesn't work fine on codeforces problem pages, what could I do?

**Comment by smblott-github** [comment]
@lonelam... What doesn't work.  Please be specific, and provide links, and steps to reproduce.

**Comment by lonelam** [comment]
thanks to reply @smblott-github 

I can't use f/F to open links on pages.


Error on a set of pages like this: http://codeforces.com/contest/752/problem/B

Chromium version:Version 55.0.2883.87 Built on Ubuntu , running on Ubuntu 16.04 (64-bit)

- Console Logs:

> Error in event handler for (unknown): TypeError: element.tagName.toLowerCase is not a function
>     at Object.getVisibleClickable (chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/content_scripts/link_hints.js:937:33)
>     at Object.getLocalHints (chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/content_scripts/link_hints.js:1071:33)
>     at chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/content_scripts/link_hints.js:127:99
>     at Object.onLoaded (chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/lib/settings.js:55:16)
>     at chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/content_scripts/link_hints.js:124:27
>     at Object.DomUtils.documentReady (chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/lib/dom_utils.js:23:18)
>     at Object.getHintDescriptors (chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/content_scripts/link_hints.js:122:23)
>     at linkHintsMessage (chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/content_scripts/vimium_frontend.js:336:50)
>     at chrome-extension://dbepggeogbaibhgnhhndojpepiihcmeb/content_scripts/vimium_frontend.js:370:99

**Comment by smblott-github** [comment]
Thanks for the report, @lonelam.  Fixed in #2385.

**Comment by philc** [comment]
For posterity, I reviewed this today and the workaround in #2385 is still needed. I noticed that the tagName issue only manifests in Chrome. In Firefox, element.tagName always returns a string.
